src/plans/D3MPlanBase.t.sol

```diff
@@ -0,0 +1,135 @@
+// SPDX-FileCopyrightText: © 2021-2022 Dai Foundation <www.daifoundation.org>
```

Comment:
Consider adopting the `DM3Plan.t.base.sol` pattern in the base tests (`pool` and `plan`) instead of `D3MPlanBase.t.sol` and `D3MPoolBase.t.sol`

Reply:
Hmm what's the advantage?

Reply:
with the new `tests` tree setup I think it's fine leaving as it is.